### ISSUE TITLE
Bad markup in doc/web2/howto.object-traversal

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4202]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4202|
|Type|enhancement|
|Created|2010-01-08 20:41:02Z|

Two problems with the markup in this file:

 1. A number of `<pre>` blocks are marked up as `<pre class="literal-block">`.  This class is not mentioned anywhere in the lore docs that I can find.  I think it should be `<pre class="python">` (all of the mis-marked blocks contain python code).
 1. The `<pre>` block starting on line 333 has `<` and `>` signs, where it should probably use `&lt;` and `&gt;`.

Attachments:

* [4202_bad_markup_fix.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/4df/4df7adde87b80f10c7a5d9e3b038222a068bfc5a/1cbed2154e309f66fdd7615831c05b20d052290a.patch) (1301 bytes) - added by khorn on 2010-01-08 20:42:12Z - FIx for markup issues in doc/web2/howto/object-traversal

<details><summary>Searchable metadata</summary>

```
trac-id__4202 4202
type__enhancement enhancement
reporter__khorn khorn
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1262983262000000 1262983262000000
changetime__1263409568000000 1263409568000000
version__None None
owner__ 
cc__khorn cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_1"></a>|@khorn removed owner|
|-|-|

patch attached, please review

**Comment by twisted-trac** [comment]
|<img alt="gas's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|gas set owner to @khorn|
|-|-|

applied patch to object-traversal.xhtml - ran lore and fixed output issue. looks good

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_3"></a>|@khorn removed owner|
|-|-|

Can somone with commit rights commit this?

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Yep.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [27977]) Apply 4202_bad_markup_fix.patch: Fix documentation markup issues with pre blocks.

Author: khorn
Reviewer: gas, thijs
Fixes: [#4202](https://github.com/twisted/twisted/issues/4202)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
